### PR TITLE
Add menu

### PR DESCRIPTION
**Pull request by thesuss**
PT Story: 
https://www.pivotaltracker.com/story/show/131070327

Changes proposed in this pull request:
- Menus can be added with title 
- Route for creating menu

To keep the PR size down we split add dishes to menu into another feature. 
(https://www.pivotaltracker.com/story/show/131350845)

What I have learned working on this feature: 
- Given/visit steps needs no expect-line
- Pair programming (ping pong) is efficient and fun (you told us, but now it's experienced)
- Basic steps are composed with ease and no longer with hesitation
- Controllers are named in plural

Screenshots: 
skarmavbild 2016-09-29 kl 10 54 33

Ready for review!

### CODE REVIEW

File: app/controllers/menu_controller.rb [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
   end
+
+  def new
+    @menu = Menu.new
+  end
+
+  def create
+    @menu = Menu.create(menu_params)
+    flash[:notice] = "Successfully added menu"
+    redirect_to menu_index_path(@menu)
```
Comment (tochman) [inline app/controllers/menu_controller.rb:12]: No need to redirect. We prefer to use render :index if that's what you want.

File: app/controllers/menu_controller.rb [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+  end
```
Comment (tochman) [inline app/controllers/menu_controller.rb:13]: I would like to see a fallback if the menu can not be saved.

File: app/views/menu/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.row
+  .column-md-12
+    %h1 add menu
+
+    = form_for @menu, url: {action: 'create'}, html: {class: 'add_menu'} do |f|
```
Comment (tochman) [inline app/views/menu/new.html.haml:5]: Don't think you need the url

File: features/add_menu.feature [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  Then I should see:
+    | content    |
+    | add menu   |
+  And I should see "create" button
+
+Scenario: Adding a menu
+  Given I am on the "add menu" page
+  When I fill in "title" with "Awesome Menu"
+  And I click the "create" button
+  Then I should see "Successfully added menu"
```
Comment (tochman) [inline features/add_menu.feature:16]: I would like to see a sad path test.

File: app/views/menu/new.html.haml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.row
+  .column-md-12
+    %h1 add menu
```
Comment (tochman) [inline app/views/menu/new.html.haml:3]: Small letters only?

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/controllers/menu_controller.rb (hunk 1 above)
Comment (thesuss) [reply app/controllers/menu_controller.rb:12]: Fixed, thanks.

File: app/controllers/menu_controller.rb (hunk 2 above)
Comment (thesuss) [reply app/controllers/menu_controller.rb:13]: Yes, right. Of course.

File: app/views/menu/new.html.haml (hunk 5 above)
Comment (thesuss) [reply app/views/menu/new.html.haml:3]: Fixed both

File: app/views/menu/new.html.haml (hunk 3 above)
Comment (thesuss) [reply app/views/menu/new.html.haml:5]: It seems like we do.

File: features/add_menu.feature (hunk 4 above)
Comment (thesuss) [reply features/add_menu.feature:16]: Done

File: app/views/menu/new.html.haml (hunk 3 above)
Comment (thesuss) [reply app/views/menu/new.html.haml:5]: Fixed =)

**Pull request merged**